lib/localisation/locales.ts

```diff
@@ -1,2 +1,9 @@
 // ! Remember to modify the middleware.ts file as you add or remove locales here
```

Comment:
Also, would be good to remove this comment as well because we are using `SUPPORTED_LOCALES` in the matcher in `nextIntlMiddleware.ts` now

Reply:
removed thanks [0208eac](https://github.com/complexdatacollective/Studio/pull/35/commits/0208eacdd8ff789cd73ba33073229320f18896a4)